cli/dcoscli/marathon/main.py

```diff
@@ -931,6 +931,20 @@ def pod_update(self, pod_id, properties, force):
         emitter.publish('Created deployment {}'.format(deployment_id))
         return 0
 
+    def _marathon_client_with_pods(self):
```

Comment:
this doesn't seem right. the client is the same. the pods command just needs a specific feature to exist